src/librustc_data_structures/obligation_forest/mod.rs

```diff
@@ -1,4 +1,3 @@
-//! The `ObligationForest` is a utility data structure used in trait
```

Comment:
Looks like this line was accidentally deleted?

Also, the big comment at the top of the file is still unchanged.